pkg/api/executor/grpc_adapter.go

```diff
@@ -36,6 +36,9 @@ type (
 		// IsInteractivitySupported is set to true only if communication platform supports interactive Messages.
 		IsInteractivitySupported bool
 
+		// KubeConfig is the path to kubectl configuration file.
```

Comment:
Better to use something like `// KubeConfig is the slice of byte representation of kubeconfig file content`